tests/layer_tests/tensorflow_lite_tests/test_tfl_Unary.py

```diff
@@ -41,9 +41,6 @@
 
     # This op could not be converted standalone -- tries to become FlexOp (offload from tfl to tf)
     # {'op_name': 'SIGN', 'op_func': tf.math.sign},
-
-    # TF has no such standalone operation
-    # {'op_name': 'HARD_SWISH'}
```

Comment:
you may also delete Shape and try to test Rank (It may only appear in the dynamic rank models which is not supported by OV)